src/price_data.py

```diff
@@ -104,6 +106,69 @@ def _get_price_binance(self, base_asset: str, utc_time: datetime.datetime, quote
         average_price = total_cost / total_quantity
         return average_price
 
+    @misc.delayed
+    def _get_price_kraken(self, base_asset: str, utc_time: datetime.datetime, quote_asset: str, minutes_step: int = 10) -> float:
+        """Retrieve price from Kraken official REST API.
+
+        We select the data point closest to the desired timestamp (utc_time), but not newer than this timestamp.
+        For this we fetch one chunk of the trade history, starting 10 minutes before this timestamp.
+        We then walk through the history until the closest timestamp match is found.
+        Otherwise, we start another 10 minutes earlier and try again, etc. …
+        (Exiting with a warning and zero price after hitting the arbitrarily chosen offset limit of 120 minutes.)
+        If the 10 minutes-offset is already too large, recursively retry by reducing the offset step, down to 1 minute.
+
+        Documentation: https://www.kraken.com/features/api
+        """
```

Comment:
We should add args/returns documentation here.
The docstrings should be according to the [google style guide](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods). Have a look at the example on the google style guide page.

Reply:
Alright. (You should also complete the docstrings for `_get_price_binance`. :) )

Reply:
Thanks :) I added the missing documentation there (https://github.com/provinzio/CoinTaxman/commit/eee3f3ba4c98def7fae0497ce6c84e8d8745116a).